Jellyfin.Api/Controllers/LibraryStructureController.cs

```diff
@@ -81,6 +81,12 @@ public async Task<ActionResult> AddVirtualFolder(
         [FromBody] AddVirtualFolderDto? libraryOptionsDto,
         [FromQuery] bool refreshLibrary = false)
     {
+        // Windows does not allow files or folders with names that has leading or trailing spaces
+        if (name.Length != name.Trim().Length)
+        {
+            return BadRequest();
```

Comment:
i totally get why we reject that, but it would not be obvious at all for a user _why_ it failed which in this instance can be quite frustraiting as its not obvious.

Either we should just trim and use the processed input or this requires a webPR too.

Reply:
> Either we should just trim

Some user may have fancy ideas to pick such name and we should warn them instead of doing silent corrections and get "bug reports" that the spaces are ignored

Reply:
i dont think so really. If i put spaces in an JF ignores them/trims them out its better to just fail and dont tell the user why.

Either like so or we rewrite the server code to trim spaces for paths always anyway and allow the user to set the name to whatever.

Reply:
> i dont think so really. If i put spaces in an JF ignores them/trims them out its better to just fail and dont tell the user why.

Then make a web change to tell user why